generate_token_json.js

```diff
@@ -0,0 +1,78 @@
+const fs = require('fs')
+const { join } = require('path')
+
+function isHyphenated(keys, length = 2) {
+  return HYPHENATED_WORDS.concat(LUMINOSITY_KEYS).includes(keys.slice(0, length).join('-'))
+}
+
+const HYPHENATED_WORDS = [
+  'z-index', 'neutral-variant', 'on-background',
+  'line-height',
+  'width-large', 'width-x-large', 'card-outline',
+  'x-small', '2x-small', '3x-small', '4x-small', '5x-small',
+  'x-large', '2x-large', '3x-large', '4x-large', '5x-large',
+  'extra-light', 'extra-bold', 'semi-bold',
+  'dialog-backdrop', 'dialog-content', 'alert-group', 'on-minus-max', 'on-minus-1', 'on-minus-2', 'on-minus-3',
+  'on-plus-min', 'on-plus-max', 'on-plus-1', 'on-plus-2', 'on-plus-3', 'on-hover', 'on-base', 'letter-spacing', 'scale-unit',
+  'plus-1', 'plus-2', 'plus-3', 'plus-max', 'minus-1', 'minus-2', 'minus-3', 'minus-max', 'input-height'
+]
+
+const LUMINOSITY_KEYS = [...Array(101).keys()].map((luminosity) => `luminosity-${luminosity}`)
```

Comment:
What does the Array(101) do? It's not obvious from this code what that means

Reply:
I added a comment to explain things